### ISSUE TITLE
Commit 5155d09 breaks file access via Page route when the configuration option cache: gzip is set to true

### ISSUE THREAD

**Issue by maxlysenko** [getgrav/grav#723]
Hi,
Commit 5155d097653b062f1439f680c7070081491a6bab breaks file access via Page route.
For example I have the following folder structure:
```
user
  pages
    01.home
      default.md
    02.features
      01.first
        default.md
      02.second
        vendor
          holder.js
        default.md
        feature02.svg
```
Then the `default.md` file inside `user/pages/02.features/02.second/` contains the following lines:
```html
<script src="/grav/features/second/vendor/holder.js" type="text/javascript" ></script>
...
<img src="second/feature02.svg" alt="Feature 02" title="Feature 02" width="180" height="180">
```
Before it worked, but after this commit access time when loading page increased up to 10s and the files are loaded with garbled content.

_Update:_ The problem appears when the configuration option `cache: gzip` is set to `true`
```yaml
cache:
  gzip: true
```

**Comment by rhukster** [comment]
This commit was to address extra spaces that were causing corruption in downloading zip files.

Can you test something for me?  Put a zip in one of your folders an test downloading it with the full path in the URL.  Something like: `/grav/features/second/test.zip`  it will download fine, but try **unzipping**.  That should expose the corruption issue.  Then try and replace the `ob_clean();` with:

```
ob_start("ob_gzhandler");
ob_start();
```

See if that helps your situation?

If it does i need to add a little logic with a gzip check.

**Comment by maxlysenko** [comment]
Hello Andy, I've done the tests that you suggested and here are the tests results with the option `cache: gzip:` set to `true`.
* without changing any lines in `Utils.php` zip archive (`test_archive.zip`) is downloaded fine but it's kind of corrupted as you suggested. In fact it's a gzipped zip, when I opened downloaded zip file with 7-zip inside there is a `test_archive` file which is the original zip archive.
* When I changed `ob_clean();` to
```php
ob_start("ob_gzhandler");
ob_start();
```
I've got an exception 
```
Whoops\Exception\ErrorException thrown with message "ob_start(): output handler 'ob_gzhandler' cannot be used twice"
```
* When I changed `ob_clean();` to `ob_start();` all is fine: zip file opens correctly and all page files are loaded correctly.

Then I've set `cache: gzip: false` and anyway all is working correctly: zip opens correctly after downloading and page resources are loaded fine. So for me in these simple tests changing `ob_clean();` to `ob_start();` solves the issue both with `cache: gzip: true` and with `cache: gzip: false`.

_These were the test I've done after seeing your reply in my email._

As for replacing `ob_clean();` with
```php
if (Grav::instance()['config']->get('system.cache.gzip')) {
    // Flush gzhandler buffer if gzip setting was enabled.
    ob_end_clean();
}
ob_clean();
```
this solves the issue for me both with `cache: gzip: true` and with `cache: gzip: false`.

Thank you.
